src/main/java/sg/ncl/MainController.java

```diff
@@ -823,9 +829,10 @@ public String validateDetails(
 
                 teamFields.put("name", signUpMergedForm.getTeamName().trim());
                 teamFields.put("description", signUpMergedForm.getTeamDescription().trim());
-                teamFields.put("website", signUpMergedForm.getTeamWebsite().trim());
+                teamFields.put(WEBSITE, signUpMergedForm.getTeamWebsite().trim());
                 teamFields.put("organisationType", signUpMergedForm.getTeamOrganizationType());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "organisationType" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)